packages/docs-site/src/library/pages/components/breadcrumbs.md

```diff
@@ -151,7 +151,7 @@ Applications will often use a library such as `React Router` to generate links b
     Type: 'Breadcrumb[] ',
     Required: 'True',
     Default: '',
-    Description: 'An array of objects that must least contain a label. If no custom component is provided then provide a href, otherwise provide the Component and the associated property to create a link',
+    Description: 'An array of objects that must at least contain a label. If no custom component is provided then provide an href, otherwise provide the Component and the associated property to create a link.',
```

Comment:
Is it right to have "an href" rather than "a href"?

Reply:
Unsure on this. I've always pronounced it "ach - ref" which to me means it reads "an href". What do you think?

Reply:
I'd say 'a href'. Check with our content person to get the official view

Reply:
Checked and it should be AN href.